File: dialogs/DialogAbout.py
from PyQt4.QtGui import *
from PyQt4.QtCore import *
from PyQt4.QtSvg import *


class DialogAbout(QDialog):
    def __init__(self, parent=None):
        super(DialogAbout, self).__init__(parent)

        self.setWindowTitle("About")

        vbox = QVBoxLayout()

        #px = QPixmap("res/icons/icon_complex64.png")

        #labelImg = QLabel()
        #labelImg.setPixmap(px)
        #labelImg.setAlignment(Qt.AlignCenter)

        #vbox.addWidget(labelImg)

        label0 = QLabel(_("<b>Jerry</b><br><br>"+
                        "Version 1.02<br><br>"+
                        "Copyright © 2014, 2015 Dominik Klein<br>"+
                        "licensed under GNU GPL 3<br><br>"+
                        "<b>Credits</b><br><br>"+
                        "Stockfish Chess Engine<br>"+
                        "by the Stockfish-Team<br><br>"
                        "python-chess library<br>"+
                        "by Niklas Fiekas<br><br>"+
                        "'VARIED.BIN' opening book<br>"+
                        "by Heinz van Saanen<br><br>"+
                        "Merida piece images<br>"+
                        "from pychess/pychess-team<br><br>"+
                        "all licensed under GNU GPL 3"))
        label0.setAlignment(Qt.AlignCenter)
        vbox.addWidget(label0)

        self.setLayout(vbox)
        self.resize(300, 150)

File: dialogs/dialog_about.cpp
/* Jerry - A Chess Graphical User Interface
 * Copyright (C) 2014-2016 Dominik Klein
 * Copyright (C) 2015-2016 Karl Josef Klein
 *
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License
 * as published by the Free Software Foundation; either version 2
 * of the License, or (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */

#include "dialog_about.h"
#include <QVBoxLayout>
#include <QLabel>

DialogAbout::DialogAbout(QWidget *parent, QString version) :
    QDialog(parent)
{

    this->setWindowTitle(this->tr("About"));
    QVBoxLayout *vbox = new QVBoxLayout();
    QHBoxLayout *hbox = new QHBoxLayout();
    QString info = QString("<br><b>Jerry</b><br><br>" \
                              "Version ");
    info.append(version).append("<br>");
    info.append("Copyright © 2014 - 2019<br>" \
                              "Dominik Klein<br>" \
                              "licensed under GNU GPL 2<br><br>" \
                              "<b>Credits</b><br><br>" \
                              "Stockfish Chess Engine<br>" \
                              "by the Stockfish-Team<br><br>" \
                              "'VARIED.BIN' opening book<br>" \
                              "by Marc Lacrosse/Jose-Chess Tool<br><br>" \
                              "Piece Images<br>" \
                              "from Raptor Chess Interface<br><br>" \
                              "all licensed under GNU GPL 2<br>");
    QLabel *labelText = new QLabel(info);
    labelText->setAlignment(Qt::AlignCenter);

    QFontMetrics f = this->fontMetrics();
    int m = f.width("m");
    vbox->addWidget(labelText);
    hbox->addSpacing(m*5);
    hbox->addLayout(vbox);
    hbox->addSpacing(m*5);
    this->setLayout(hbox);

}
